experiment/logviewer/gcsreader/parsing_test.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"bytes"
+	"io/ioutil"
+	"path/filepath"
+	"regexp"
+	"testing"
+)
+
+func Test_MatchSingleLine(t *testing.T) {
+	data, err := ioutil.ReadFile(filepath.Join("testdata", "kube-apiserver-cut.log"))
```

Comment:
I think it could be slightly better if we add few line text constant in this file instead of adding testdata dir. The main advantage is that we won't need to switch files to understand what this file contain and what the expected result is.